lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -457,6 +482,25 @@ def __init__(self, message="Alert rule is not well formed"):
         super().__init__(self.message)
 
 
+class LokiPushApiEndpointDeparted(EventBase):
+    """Event emitted when Loki departed."""
+
+    pass
```

Comment:
`pass` is redundant when you already have a docstring.

Reply:
Thanks, I forgot to deleted!